packages/web-pixels-extension/README.md

```diff
@@ -0,0 +1,40 @@
+# Web Pixels Extension
+
+Provides an `extend` method to register a Web Pixels Extension.
+
+## Installation
+
+```bash
+$ yarn add @shopify/web-pixels-extension
+```
+
+## Usage
+
+Registering a Web Pixels Extension:
+
+```js
+import {register} from '@shopify/web-pixels-extension';
+
+register((
+  {configuration, analytics, client}) => {
```

Comment:
Why was `client` chosen as the name for this field? I find it a little confusing, personally, since it's actually talking about a bridge to what we often refer to as the `host` (basically, the opposite of `client`).

Reply:
I think `browser` or `page` is more evocative of what you are getting at with `client`, in that case. If it were me, I might just put all those objects as siblings of `configuration`/ `analytics`, if there aren't too many of them (that's basically what we do with checkout extensions, where every <thing> an extension can do is a top-level object in this "API object")

Reply:
cool. let's do browser.